.devcontainer/Dockerfile

```diff
@@ -55,4 +46,8 @@ RUN \
     echo "\n# add local python programs to PATH\nexport PATH=$PATH:$HOME/.local/bin\n" >> $HOME/.bashrc && \
     # enable some useful aliases
     sed -i -e "s/#alias/alias/" $HOME/.bashrc && \
-    echo "\n# skip sudo for docker commands\nalias docker='sudo docker'" >> $HOME/.bashrc
+    pip install cookiecutter

```

Comment:
>cookiecutter [](http://example.com/codeflow?start=16&length=12)

why is cookiecutter needed here?

Reply:
When running some manual tests, I used iotedgedev to setup a solution.